crates/duty-tracker/src/contract_manager.rs

```diff
@@ -948,7 +948,7 @@ async fn execute_duty(
                             .expect("failed to construct sighash");
                         let signature = s2_client
                             .general_wallet_signer()
-                            .sign(&sighash.to_byte_array())
+                            .sign(&sighash.to_byte_array(), None)
```

Comment:
This is odd given that this commit doesn't seem to pin a new version of the s2 client. When did the s2 change get made?